src/components/note/__internal__/note.component.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Editor, EditorState } from 'draft-js';
+import {
+  StyledNote,
+  StyledNoteContent,
+  StyledInlineControl,
+  StyledTitle,
+  StyledFooter,
+  StyledFooterContent
+} from './note.style.js';
+import StatusWithTooltip from './status-with-tooltip';
+
+const Note = ({
+  noteContent = EditorState.createEmpty(),
+  width,
+  inlineControl,
+  title,
+  name,
+  createdDate,
+  status,
+  ...rest
+}) => {
+  const [showTooltip, setShowTooltip] = useState(false);
+
+  const renderStatus = () => {
+    if (!status || !status.text || !status.text.length) {
+      return null;
+    }
+
+    const { text, timeStamp } = status;
+
+    return (
+      <StyledFooterContent data-component='note-status'>
```

Comment:
Should it be like: 
```suggestion
      <StyledFooterContent data-component="note">
```

Reply:
No the parent component already has that data tag I think `note-status` is okay, there is also a tag on the component itself so should be enough for cypress to target